### ISSUE TITLE
Finish integrating semantic changes to a11y scans

### ISSUE THREAD

**Issue by gbinal** [18F/domain-scan#111]
Right now, the following edits are manually made during the a11y scan process.  We should go through and change the scripts and scans to address these so that I no longer need to manually make them:  

* Alt Text => Missing Image Description  - [issue](https://github.com/18F/pulse/issues/536)
* add http:// to canonical domains - [issue](https://github.com/18F/pulse/issues/565)
* "errors" - "initial findings" - [issue](https://github.com/18F/pulse/issues/609)
* agency -> Agency - [issue](https://github.com/18F/pulse/pull/583)

**Comment by gbinal** [comment]
It looks like the 1st, 3rd, and 4th of these was addressed in #115.  I'll go back in and try to resolve the last of them.

**Comment by konklone** [comment]
@gbinal - Just to confirm, is this still outstanding? If so, do you want to keep it open?
